src/features/metadata-instance-editor/CascadePolicy.js

```diff
@@ -10,22 +13,29 @@ import messages from './messages';
 import './CascadePolicy.scss';
 
 const COMMUNITY_LINK = 'https://support.box.com/hc/en-us/articles/360044195873-Cascading-metadata-in-folders';
-
+const AI_LINK = 'https://www.box.com/ai';
+const PRICING_LINK = 'https://www.box.com/pricing';
 type Props = {
     canEdit: boolean,
+    canUseAIFolderExtraction?: boolean,
+    isAIFolderExtractionEnabled: boolean,
```

Comment:
should `canUseAIFolderExtraction` be required since `isAIFolderExtractionEnabled` and `onAIFolderExtractionToggle` are required?

the prop has a default set in `MetadataInstanceEditor` and `Instances` so should be okay to have required here